doc/nrf/protocols/matter/end_product/certification.rst

```diff
@@ -330,3 +332,26 @@ This variant is meant for Matter components that use Matter software that was al
    :alt: Matter certification by similarity overview
 
    Matter certification by similarity overview
+
+.. _ug_matter_device_certification_matter_samples:
+
+Certification status of Matter samples
+**************************************
+
+The |NCS| includes several :ref:`matter_samples` that are example implementations of Matter devices.
+The samples are maintained to fulfill Matter certification requirements as much as possible, however they did not get official Matter compliance certificates.
+You can use them as a reference for creating an own application, however it does not guarantee that the application will pass the Matter certification.
+
+The Matter certification program currently does not provide any form of platform certification, so the product maker is solely reponsible for ensuring that the application used fulfills all Matter certification requirements.
+Before you start applying for the Matter certification, you should make sure that the application configuration generated using :file:`.zap` file is compatible with the PICS generated for the certification purpose.
+You can find the information about the specification and how to select PICS for you product on the `CSA Matter Resource Kit`_ page, in the `Specification` section.
+Especially, make sure that:
+
+* ``FeatureMap`` attributes for all clusters match the desired configuration selected in PICS and all other cluster requirements related to the specific ``FeatureMap`` value are met.
+* ``ClusterRevision`` attributes for all clusters match the value of Matter stack version that you want to use for you application certification.
+
+To read about how to configure your application using the ZAP Tool, see `ug_matter_creating_accessory`_ page.
+
+Before you request the official product `ug_matter_device_certification_testing`_ from ATL, it is recommended to perform the in-house product testing.
+The initial verification allows to spot majority of problems that could lead to further certification failure, and in result safe your time and money spend on official certification in ATL laboratory.
```

Comment:
```suggestion
This testing allows you to avoid most problems that could lead to further certification failure, meaning it saves time and money that would be spent on official certification in ATL laboratory.
```

Reply:
I'm not sure whether we should use "avoid" word. What I wanted to write is that the intention of testing in-house is to detect firmware bug earlier and be able to fix it before spending money on testing in ATL.

Reply:
In that case, something like this is probably better:
```suggestion
This testing can help you detect problems that could lead to certification failure, meaning it saves time and money that would be spent on official certification in ATL laboratory.
```